app/client/src/components/pages/StateTribal.Routes.StateTribalTabs.js

```diff
@@ -120,7 +121,7 @@ function StateTribalTabs() {
 
     return (
       <div>
-        <h2>
+        <h2 className={h2Styles}>
```

Comment:
```suggestion
        <h2 css={h2Styles}>
```

This is causing console errors and the styles aren't applied.

Reply:
Oof, good catch!